src/Domain/FunctionsOfTime/PiecewisePolynomial.cpp

```diff
@@ -91,7 +90,7 @@ void PiecewisePolynomial<MaxDeriv>::update(
           << func.back().size() << ") in the PiecewisePolynomial.");
   }
 
-  func[MaxDeriv] = std::move(updated_max_deriv);
+  func[MaxDeriv] = updated_max_deriv;
```

Comment:
I changed this line, and the input argument updated_max_deriv (which used to be a DataVector passed by value and is now a `const DataVector &`) because clang-tidy complained about passing by value.  If you want, I can restore the std::move, restore the pass-DataVector-by-value, and add a NOLINT for clang-tidy.

Reply:
Yeah, I have no idea what clang-tidy is thinking here.  I can reproduce the warnings locally, and it is literally warning that the variable should be made const, and when you do that warning that the variable should not be const.  Just find somewhere to put a NOLINT to shut it up.